lib/ReactGridLayout.jsx

```diff
@@ -152,11 +166,33 @@ var ReactGridLayout = React.createClass({
     this.setState({width: width});
   },
 
-  onDragStart(i, e, {element, position}) {
-    // nothing
+  /**
+   * When dragging starts
+   * @param {Number} i Index of the child
+   * @param {Number} x X position of the move
+   * @param {Number} y Y position of the move
+   * @param {Element} element The current dragging DOM element
+   * @param {Object} position Drag information
+   */
+  onDragStart(i, x, y, {e, element, position}) {
+    var layout = this.state.layout;
+    var l = utils.getLayoutItem(layout, i);
+    if (this.props.onDragStart(layout, this.state.layout, l, null, {e, element, position}) === false) {
```

Comment:
Yeah, I think e.preventDefault would be the way to go since I'm not necessarily a fan of returning false, and as of React .12 you must call preventDefault to stop.
It looks like we could do it with mouseDown event since that is what react-draggable is using, however we may need to mod the handleStartDrag to actually do the preventDefault check.